applications/opencost/2.3.1/metadata.yaml

```diff
@@ -3,6 +3,8 @@ displayName: OpenCost
 description: OpenCost provides cost monitoring and resource optimization for Kubernetes workloads.
 category:
   - monitoring
+requiredDependencies:
```

Comment:
maybe not a hard dep bc users can update config to point to their own prom

Reply:
done